snail-kotlin/src/main/java/com/compass/snail/extensions/TextViewExtensions.kt

```diff
@@ -0,0 +1,22 @@
+//  Copyright © 2017 Compass. All rights reserved.
+
+package com.compass.snail.extensions
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.TextView
+import com.compass.snail.Observable
+import com.compass.snail.Replay
+
+inline val TextView.textChanged: Observable<String>
+    get() {
+        val observable = Replay<String>(1)
+        this.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) {}
+            override fun onTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) {
```

Comment:
good point! I was curious what you thought here. We can definitely add another extension for that.

Also @gray419 I could not figure out how to unit test this. The whole android context for the TextView stumped me. If you have thoughts there, that would be rad.